### ISSUE TITLE
Install failed using Node v10

### ISSUE THREAD

**Issue by igor-sky** [bnoordhuis/node-unix-dgram#48]
```nan``` versions is outdated, newer versions of ```nan``` does not have that issue

```
npm i unix-dgram@2.0.0

> unix-dgram@2.0.0 install /private/tmp/qqq/node_modules/unix-dgram
> node-gyp rebuild

  CXX(target) Release/obj.target/unix_dgram/src/unix_dgram.o
In file included from ../src/unix_dgram.cc:5:
In file included from ../../nan/nan.h:190:
../../nan/nan_maybe_43_inl.h:88:15: error: no member named 'ForceSet' in 'v8::Object'
  return obj->ForceSet(GetCurrentContext(), key, value, attribs);
         ~~~  ^
```

**Comment by brettkiefer** [comment]
@igor-sky Fixed in 2.0.2, right?

**Comment by bnoordhuis** [comment]
Yep, upgrade and all should be well.
